code/_onclick/hud/hologram.dm

```diff
@@ -0,0 +1,202 @@
+/datum/hud/proc/hologram_hud(var/ui_style='icons/mob/screen1_White.dmi')
+	src.adding = list()
+	src.other = list()
+	ui_style = 'icons/mob/screen1_White.dmi'
+
+	var/obj/abstract/screen/using
+	var/obj/abstract/screen/inventory/inv_box
+
+	using = getFromPool(/obj/abstract/screen)
+	using.name = "act_intent"
+	using.dir = SOUTHWEST
+	using.icon = ui_style
+	using.icon_state = (mymob.a_intent == I_HURT ? "harm" : mymob.a_intent)
+	using.screen_loc = ui_acti
+	src.adding += using
+	action_intent = using
+
+//intent small hud objects
+	var/icon/ico
+
+	ico = new(ui_style, "black")
+	ico.MapColors(0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0, -1,-1,-1,-1)
+	ico.DrawBox(rgb(255,255,255,1),1,ico.Height()/2,ico.Width()/2,ico.Height())
+	using = getFromPool(/obj/abstract/screen,src)
+	using.name = "help"
+	using.icon = ico
+	using.screen_loc = ui_acti
+	using.layer = HUD_ABOVE_ITEM_LAYER
+	src.adding += using
+	help_intent = using
+
+	ico = new(ui_style, "black")
+	ico.MapColors(0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0, -1,-1,-1,-1)
+	ico.DrawBox(rgb(255,255,255,1),ico.Width()/2,ico.Height()/2,ico.Width(),ico.Height())
+	using = getFromPool(/obj/abstract/screen,src)
+	using.name = "disarm"
+	using.icon = ico
+	using.screen_loc = ui_acti
+	using.layer = HUD_ABOVE_ITEM_LAYER
+	src.adding += using
+	disarm_intent = using
+
+	ico = new(ui_style, "black")
+	ico.MapColors(0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0, -1,-1,-1,-1)
+	ico.DrawBox(rgb(255,255,255,1),ico.Width()/2,1,ico.Width(),ico.Height()/2)
+	using = getFromPool(/obj/abstract/screen,src)
+	using.name = "grab"
+	using.icon = ico
+	using.screen_loc = ui_acti
+	using.layer = HUD_ABOVE_ITEM_LAYER
+	src.adding += using
+	grab_intent = using
+
+	ico = new(ui_style, "black")
+	ico.MapColors(0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0, -1,-1,-1,-1)
+	ico.DrawBox(rgb(255,255,255,1),1,1,ico.Width()/2,ico.Height()/2)
+	using = getFromPool(/obj/abstract/screen,src)
+	using.name = "harm"
+	using.icon = ico
+	using.screen_loc = ui_acti
+	using.layer = HUD_ABOVE_ITEM_LAYER
+	src.adding += using
+	hurt_intent = using
+
+//end intent small hud objects
+
+	using = getFromPool(/obj/abstract/screen)
+	using.name = "mov_intent"
+	using.dir = SOUTHWEST
+	using.icon = ui_style
+	using.icon_state = (mymob.m_intent == "run" ? "running" : "walking")
+	using.screen_loc = ui_movi
+	src.adding += using
+	move_intent = using
+
+	using = getFromPool(/obj/abstract/screen)
+	using.name = "drop"
+	using.icon = ui_style
+	using.icon_state = "act_drop"
+	using.screen_loc = ui_drop_throw
+	using.layer = HUD_BASE_LAYER
+	src.adding += using
+
+	init_hand_icons(ui_style)
+
+	#define ui_swaphand_hologram1	"CENTER-1:16,SOUTH:5"
+	#define ui_swaphand_hologram2	"CENTER:16,SOUTH:5"
+	#define ui_equip_hologram		"CENTER:16,SOUTH:5"
```

Comment:
You're not cleaning this define up it seems.

Reply:
Fixed.